stable/mongodb/README.md

```diff
@@ -43,36 +43,52 @@ The command removes all the Kubernetes components associated with the chart and
-| `livenessProbe.periodSeconds`       | How often to perform the probe                                                               | 10                                                       |
-| `livenessProbe.timeoutSeconds`      | When the probe times out                                                                     | 5                                                        |
-| `livenessProbe.successThreshold`    | Minimum consecutive successes for the probe to be considered successful after having failed. | 1                                                        |
-| `livenessProbe.failureThreshold`    | Minimum consecutive failures for the probe to be considered failed after having succeeded.   | 6                                                        |
-| `readinessProbe.initialDelaySeconds`| Delay before readiness probe is initiated                                                    | 5                                                        |
-| `readinessProbe.periodSeconds`      | How often to perform the probe                                                               | 10                                                       |
-| `readinessProbe.timeoutSeconds`     | When the probe times out                                                                     | 5                                                        |
-| `readinessProbe.successThreshold`   | Minimum consecutive successes for the probe to be considered successful after having failed. | 1                                                        |
-| `readinessProbe.failureThreshold`   | Minimum consecutive failures for the probe to be considered failed after having succeeded.   | 6                                                        |
+|         Parameter                       |             Description                                                                      |                         Default                          |
+|-----------------------------------------|----------------------------------------------------------------------------------------------|----------------------------------------------------------|
+| `image.registry`                        | MongoDB image registry                                                                       | `docker.io`                                              |
+| `image.repository`                      | MongoDB Image name                                                                           | `bitnami/mongodb`                                        |
+| `image.tag`                             | MongoDB Image tag                                                                            | `{VERSION}`                                              |
+| `image.pullPolicy`                      | Image pull policy                                                                            | `Always` if `imageTag` is `latest`, else `IfNotPresent`  |
```

Comment:
The default value column should just say `Always`

Reply:
done